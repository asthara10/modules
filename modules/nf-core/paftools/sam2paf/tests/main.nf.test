nextflow_process {

    name "Test Process PAFTOOLS_SAM2PAF"
    script "../main.nf"
    process "PAFTOOLS_SAM2PAF"

    tag "modules"
    tag "modules_nfcore"
    tag "paftools"
    tag "paftools/sam2paf"

    test("test_paftools_sam2paf") {


        when {
            process {
                """
                
                input[0] = [
        [id:'test'],
        file(params.test_data['homo_sapiens']['scramble']['bam'],
        checkIfExists: true)
    ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
