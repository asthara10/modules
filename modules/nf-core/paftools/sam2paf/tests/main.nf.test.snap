{
    "versions": {
        "content": [
            [
                "versions.yml:md5,ae0342a5ff0006c183531b23d9fb5d4f"
            ]
        ],
        "timestamp": "2023-11-24T10:36:41.028443"
    }
}